src/main/io/beeper.c

```diff
@@ -241,7 +241,7 @@ void beeper(beeperMode_e mode)
         mode == BEEPER_SILENCE || (
             (beeperConfig()->beeper_off_flags & BEEPER_GET_FLAG(BEEPER_USB))
             && getBatteryState() == BATTERY_NOT_PRESENT
-        )
+        ) || (IS_RC_MODE_ACTIVE(BOXBEEPERMUTE) && !IS_RC_MODE_ACTIVE(BOXBEEPERON))
```

Comment:
Not sure about this - having the 'beeper mute' only work selectively on beepers except for some (BEEPER_RX_SET) under some conditions is a logical flaw. I don't think this should be checking `BOXBEEPERON` here.

Reply:
@mikeller , As stated in the original user request for this feature (#10204):
> Activating the buzzer manually (with the current buzzer mode) should override the buzzer silence mode.

the `&& !IS_RC_MODE_ACTIVE(BOXBEEPERON)` is introduced here. This is condition where we should override active 'Beeper mute' mode. This is covered in the real life situation where user lost his/her craft, but still has a control link and when 'Beeper mute' mode is active, he/she is able to activate the buzzer via switch. It's just an safety measure in this kind of situation. If we remove this condition, it would be tricky to force activate a beeper when we have 'Beeper mute' mode active. In that case, the only solution would be to have another Fail safe switch which forces the craft into the FS mode and deactivates the 'Beeper mute' and only then a user can activate a beeper via switch. But it's too complicated and it costs one channel more (for the FS mode). 

This feature is hard tested on the bench and on a field with all possible combinations (warnings, init's, FS, etc) by original author of this request and me. It has no side effect (when is active).

Reply:
@dkustec: I don't follow:

> This is covered in the real life situation where user lost his/her craft, but still has a control link and when 'Beeper mute' mode is active, he/she is able to activate the buzzer via switch.

This implies that the failure is such that the user still has got control over one AUX channel (`BOXBEEPERON`), but not another (`BOXBEEPERMUTE`) - this seems to be very far fetched.

To me, this is just unlogical behaviour that will end up in endless user questions.

Reply:
@mikeller , yes, when I rethink about it - it makes sense what you talking about. It can be be bypassed if we have on one switch `BOXBEEPERON` and `BOXBEEPERMUTE`. If we want to activate a beeper, the same switch can temporary disable `BOXBEEPERMUTE` and enable `BOXBEEPERON`. 

> that will end up in endless user questions.

And this is we want to avoid. So I will remove this condition. 
Thanks for clearing things out.

Reply:
The idea behind the manual override is as follows:
I have my beeper together with a pre-arm on a momentary switch. When I flick that switch I always want the beeper to sound. Even if beeper mute is on. Beeper mute is on a separate switch.
Why? Its a warning for both myself and others that I'm about to arm my quad. So I don't want to first have to disable beepermute before I can beep.

Another use case; you crashed far away in a field or in bushes, so you want to beep to find it. However, when you crashed or during the search lowbat or something comes on. The way I see it is that I only want to have it beeping for nothing, so I want to mute the lowbat beeps. Do I still want my manual beeps in order to find it? Yes please! If you then first have to disable mute it would be confusing in my opinion.

Basically; the way it was is perfect. I have flown many packs with the function. And yes; above mentioned cases have occurred and I was glad to have it.